pycbc/filter/matchedfilter.py

```diff
@@ -1831,16 +1831,55 @@ def followup_event_significance(ifo, data_reader, bank,
     onsource_end += coinc_threshold
 
     # Calculate how much time needed to calculate significance
-    trim_pad = (data_reader.trim_padding * data_reader.strain.delta_t)
-    bdur = int(lookback + 2.0 * trim_pad + length_in_time)
-    if bdur > data_reader.strain.duration * .75:
-        bdur = data_reader.strain.duration * .75
+    trim_pad = data_reader.trim_padding * data_reader.strain.delta_t
+    buffer_duration = lookback + 2 * trim_pad + length_in_time
+    buffer_samples = bank.round_up(int(buffer_duration * bank.sample_rate))
```

Comment:
I note that this is not rounding up the nearest integer, but to the nearest increment time (ie. to the nearest 8s). Is this intended/desired? If so, a comment might help.

Reply:
Yes, this is intended, as it does what `bank.get_template()` was doing before this change. I agree that this is a good place for a comment that explains why this nontrivial calculation looks like that. Unless @ahnitz beats me with a good suggestion, I will have a fresh look at the code to remind myself.

Reply:
Comment added, hopefully both clear and correct.